public/app/features/variables/pickers/shared/VariableInput.tsx

```diff
@@ -21,8 +22,10 @@ export class VariableInput extends PureComponent<Props> {
   };
 
   render() {
+    const { value, id, onNavigate, ...restProps } = this.props;
     return (
       <input
+        {...restProps}
```

Comment:
What props are we expecting to pass/spread here?

Reply:
It's the aria attributes. Not sure importing the explicitly is the best approach.

Reply:
I see, could VariableInput extend some HTML interface instead like extends Omit<HTMLProps<HTMLInputElement>, 'x' | 'y' | 'z'>

Reply:
Sounds like a good idea! The rest spread should still be there, right?

Reply:
Yes indeed!